apps/meteor/client/views/account/security/TwoFactorTOTP.tsx

```diff
@@ -80,20 +77,23 @@ const TwoFactorTOTP = (props: ComponentProps<typeof Box>): ReactElement => {
 		setModal(<TwoFactorTotpModal onConfirm={onDisable} onClose={closeModal} />);
 	}, [closeModal, disableTotpFn, dispatchToastMessage, setModal, t]);
 
-	const handleVerifyCode = useCallback(async () => {
-		try {
-			const result = await verifyCodeFn(authCode);
+	const handleVerifyCode = useCallback(
+		async ({ authCode }) => {
+			try {
+				const result = await verifyCodeFn(authCode);
 
-			if (!result) {
-				return dispatchToastMessage({ type: 'error', message: t('Invalid_two_factor_code') });
-			}
+				if (!result) {
```

Comment:
IMO we can remove this check, it's unnecessary.

Reply:
not really, `2fa:validateTempToken` method can return `undefined`